.github/workflows/update_template.yml

```diff
@@ -0,0 +1,39 @@
+name: Update Template
+
+on:
+  workflow_run:
+    workflows: [Release]
+    types:
+      - completed
+
+jobs:
+  release:
+    # prevents this action from running on forks
+    if: github.repository == 'sveltejs/kit'
+    name: build
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out kit repository
+        uses: actions/checkout@v2
+        with:
+          path: kit
+
+      - name: Check out template repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          ssh-key: "${{ secrets.UPDATE_TEMPLATE_SSH_KEY }}"
+          repository: kit-template-default
+          path: template   
+
+      - name: Write template
+        working-directory: ./kit/packages/create-svelte
+        run: node $DIR/update-template-repo-contents.js ./template
```

Comment:
is `$DIR` a thing or is this copy-pasted from #3451?

Reply:
copypasta! oops :)